udata/core/dataset/apiv2.py

```diff
@@ -217,3 +217,12 @@ def get(self, dataset):
             'previous_page': previous_page if page > 1 else None,
             'total': len(res),
         }
+
+
+@ns.route('/resources/<uuid:rid>/', endpoint='resource')
+class ResourceAPI(API):
+    @apiv2.doc('get_resource')
+    @apiv2.marshal_with(resource_fields)
+    def get(self, rid):
+        return get_resource(rid)
```

Comment:
I think we should add `dataset id` in return value, for ulterior api calls.

Reply:
tell me WDYT of the new implem